test/test_helper.exs

```diff
@@ -1,3 +1,3 @@
 Application.ensure_all_started(:ex_machina)
-ExUnit.start(exclude: [:skip])
+ExUnit.start(exclude: [:skip], capture_log: true)
```

Comment:
do we need capture log here or was it just useful for you personally? (ie: extraneous noise for others perhaps?)